.github/workflows/ci-conda.yml

```diff
@@ -90,12 +90,14 @@ jobs:
         run: |
           ./bootstrap
           echo "::add-matcher::.github/workflows/configure-systempackage-problem-matcher.json"
-          ./configure --enable-build-as-root --with-python=$CONDA_PREFIX/bin/python --prefix=$CONDA_PREFIX $(for pkg in $(./sage -package list :standard: --has-file spkg-configure.m4 --has-file distros/conda.txt); do echo --with-system-$pkg=force; done)
+          ./configure --enable-build-as-root --with-python=$CONDA_PREFIX/bin/python --prefix=$CONDA_PREFIX $(for pkg in $(./sage -package list :standard: --has-file spkg-configure.m4 --has-file distros/conda.txt); do if [ $pkg != "rpy2" ]; then echo --with-system-$pkg=force; fi; done)
```

Comment:
You can also use `sage -package .... --exclude PACKAGE_NAME`

Reply:
Thanks, done.